packages/kbn-plugin-generator/integration_tests/generate_plugin.test.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint-disable no-restricted-syntax */
+import { spawn } from 'child_process';
+import { resolve } from 'path';
+import util from 'util';
+import { stat } from 'fs';
+import { snakeCase } from 'lodash';
+import del from 'del';
+import { withProcRunner, ToolingLog } from '@kbn/dev-utils';
+import { createEsTestCluster } from '@kbn/test';
+import execa from 'execa';
+
+const statP = util.promisify(stat);
+const ROOT_DIR = resolve(__dirname, '../../../');
+const oneMinute = 60000;
+
+describe(`running the plugin-generator via 'node scripts/generate_plugin.js plugin-name' with default config`, () => {
+  const pluginName = 'ispec-plugin';
+  const snakeCased = snakeCase(pluginName);
+  const generatedPath = resolve(ROOT_DIR, `plugins/${snakeCased}`);
+  const collect = xs => data => xs.push(data + ''); // Coerce from Buffer to String
+
+  beforeAll(() => {
+    jest.setTimeout(oneMinute * 10);
+  });
+
+  beforeAll(done => {
+    const create = spawn(process.execPath, ['scripts/generate_plugin.js', pluginName], {
+      cwd: ROOT_DIR,
+    });
+    create.stdout.on('data', () => {
```

Comment:
Mind tossing a comment here to describe what's happening?

Reply:
You betcha!